From 596810f6a2c9214d85dce9e9300d6768a7e2536d Mon Sep 17 00:00:00 2001
From: Miklos Szegedi <miklos.szegedi@cloudera.com>
Date: Tue, 8 Nov 2016 16:24:00 -0800
Subject: [PATCH 2009/2848] CDH-44809. Disabling
 TestCapacityScheduler.testPreemptionInfo as it
 fails occasionally and we won't support the
 feature in the future.

Change-Id: If11a1f5b86a70bec761ba23338c660a911294fd1
---
 .../scheduler/capacity/TestCapacityScheduler.java  |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacityScheduler.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacityScheduler.java
index af75876..76d4ffb 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacityScheduler.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/TestCapacityScheduler.java
@@ -139,6 +139,7 @@
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 import org.mockito.Mockito;
 
@@ -1040,7 +1041,12 @@ public void testNumClusterNodes() throws Exception {
     cs.stop();
   }
 
+  /**
+   * This test is ignored due to occasional test failures.
+   * We plan not to support the feature in the future.
+   */
   @Test(timeout = 120000)
+  @Ignore
   public void testPreemptionInfo() throws Exception {
     Configuration conf = new Configuration();
     conf.setInt(YarnConfiguration.RM_AM_MAX_ATTEMPTS, 3);
-- 
1.7.9.5

